contrib/win32/win32compat/wmain_sshd.c

```diff
@@ -163,15 +165,19 @@ create_prgdata_ssh_folder()
 	char ssh_cfg_dir[PATH_MAX] = { 0, };
 	strcpy_s(ssh_cfg_dir, _countof(ssh_cfg_dir), get_program_data_path());
 	strcat_s(ssh_cfg_dir, _countof(ssh_cfg_dir), "\\ssh");
-	if (create_directory_withsddl(ssh_cfg_dir, "O:BAD:PAI(A;OICI;FA;;;SY)(A;OICI;FA;;;BA)(A;OICI;0x1200a9;;;AU)") < 0)
-		fatal("failed to create %s", ssh_cfg_dir);
+	if (create_directory_withsddl(ssh_cfg_dir, "O:BAD:PAI(A;OICI;FA;;;SY)(A;OICI;FA;;;BA)(A;OICI;0x1200a9;;;AU)") < 0) {
+		printf("failed to create %s", ssh_cfg_dir);
```

Comment:
Would you see these at all? since this logic is invoked only during sshd service startup and that happens in session 0

Reply:
We never see them. The printf statements helps us to debug quickly, when we have private binary where we call prereq_setup() during sshd startup (need not be service startup).